src/analog.jl

```diff
@@ -11,22 +8,22 @@ analog_input_configs = Dict{AbstractString,Number}(
 
 create an analog input channel, and a new task if one is not specified
 """
-function analog_input(channel::String; terminal_config="differential", range=nothing)
+function analog_input(channel::String; terminal_config::TerminalConfig=NRSE, range=nothing)
     t = AITask()
     analog_input(t, channel, terminal_config=terminal_config, range=range)
     t
 end
 
 function analog_input(t::AITask, channel::String;
-        terminal_config="differential", range=nothing)
+        terminal_config::TerminalConfig=NRSE, range=nothing)
     if range==nothing
         device::String = split(channel,'/')[1]
         range=float(analog_input_ranges(device)[end,:])
     end
     catch_error( CreateAIVoltageChan(t.th,
-            Ref(codeunits(channel),1),
-            Ref(codeunits(""), 1),
-            analog_input_configs[terminal_config],
+            pointer(channel),
+            pointer(""),
+            Cuint(terminal_config),
```

Comment:
could the the `Cuint()` caste be omitted here if `TerminalConfig` was declared to be `::Cuint` when defined?

Reply:
Will try that

Reply:
That worked.